docs/core/diagnostics/logging-tracing.md

```diff
@@ -1,110 +1,55 @@
-  - Because it's the simplest approach, it's often used for ad-hoc temporary debugging. This debug code is often never checked in to source control.
-- <xref:System.Diagnostics.Trace?displayProperty=nameWithType>
-  - Only enabled when `TRACE` is defined by adding `#define TRACE` to your source or specifying the option `/d:TRACE` when compiling.
-  - Writes to attached <xref:System.Diagnostics.Trace.Listeners>, by default the <xref:System.Diagnostics.DefaultTraceListener>.
-  - Use this API when creating logs that will be enabled in most builds.
-- <xref:System.Diagnostics.Debug?displayProperty=nameWithType>
-  - Only enabled when `DEBUG` is defined by adding `#define DEBUG` to your source or specifying the option `/d:DEBUG` when compiling.
-  - Writes to an attached debugger.
-  - On `*nix` writes to stderr if `DOTNET_DebugWriteToStdErr` or `COMPlus_DebugWriteToStdErr` is set.
-  - Use this API when creating logs that will be enabled only in debug builds.
+### Sinks
 
-### Logging events
+Most logging APIs allow log messages to be sent to different destinations called sinks. Some APIs have a large number of pre-made sinks whereas others only have a few. If no pre-made sink exists there is usually an extensibility API that will let you author a custom sink but this will take a little extra effort.
 
-The following APIs are more event oriented. Rather than logging simple strings they log event objects.
+## .NET logging APIs
 
-- [System.Diagnostics.Tracing.EventSource](./eventsource.md)
-  - EventSource is the primary root .NET Core tracing API.
-  - Available in all .NET Standard versions.
-  - Only allows tracing serializable objects.
-  - Can be consumed in-process via any [EventListener](xref:System.Diagnostics.Tracing.EventListener) instances configured to consume the EventSource.
-  - Can be consumed out-of-process via:
-    - [.NET Core's EventPipe](./eventpipe.md) on all platforms
-    - [Event Tracing for Windows (ETW)](/windows/win32/etw/event-tracing-portal)
+### ILogger
 
-- <xref:System.Diagnostics.DiagnosticSource?displayProperty=nameWithType>
-  - Included in .NET Core and as a [NuGet package](https://www.nuget.org/packages/System.Diagnostics.DiagnosticSource) for .NET Framework.
-  - Allows in-process tracing of non-serializable objects.
-  - Includes a bridge to allow selected fields of logged objects to be written to an <xref:System.Diagnostics.Tracing.EventSource>.
+For most cases, whether adding logging to an existing project or creating a new project, [ILogger](../extensions/logging.md) is a good default choice for logging API. ILogger supports fast structured logging, flexible configuration, and a collection of [common sinks](../extensions/logging-providers.md#built-in-logging-providers). ILogger can also serve as a facade over many [third party logging implementations](../extensions/logging-providers.md#third-party-logging-providers) that offer much more functionality and extensibility.
```

Comment:
```suggestion
For most cases, whether adding logging to an existing project or creating a new project, the [ILogger](../extensions/logging.md) is a good default choice. `ILogger` supports fast structured logging, flexible configuration, and a collection of [common sinks](../extensions/logging-providers.md#built-in-logging-providers). Additionally. the `ILogger` can also serve as a facade over many [third party logging implementations](../extensions/logging-providers.md#third-party-logging-providers) that offer more functionality and extensibility.
```

Reply:
I assume that period after "Additionally" was intended to be a comma?

Reply:
"The ILogger" sounds odd to me - did you mean to write "The ILogger API"?
If we were referring to an instance of ILogger I'd probably say "an ILogger", but when refering to the API as a whole I would say "ILogger" with no article. For example "ILogger is a great API", or "You should use ILogger"

Reply:
I was treating it as the interface, for example, the `ILogger` interface. Maybe that specificity would help? I fixed the suggestion, thoughts?

Reply:
I think "the ILogger interface" sounds fine. There is still "the ILogger is a good default choice" in the updated suggestion. Was that also intended to have 'interface' added?